config/deployment.yml

```diff
@@ -36,16 +36,12 @@ apiVersion: v1
 kind: Service
 metadata:
   name: python-example-service
-  annotations:
```

Comment:
These annotations are used to manage DNS using `external-dns` and set up SSL. It's not strictly required for a push to deploy example and would require additional prerequisites. 

https://www.digitalocean.com/docs/kubernetes/how-to/configure-load-balancers/#ssl-certificates
https://blog.andrewsomething.com/2019/04/04/external-dns-with-ssl-on-k8s/